browser_patches/repack-juggler.mjs

```diff
@@ -108,27 +109,48 @@ async function ensureFirefoxBuild(browserName, buildNumber, buildPlatform) {
     throw new Error(`ERROR: repack-juggler does not support ${buildPlatform}`);
   const url = util.format(urlTemplate, buildNumber);
   console.log(`Downloading ${browserName} r${buildNumber} for ${buildPlatform} - it might take a few minutes`);
-  await downloadFile(url, buildZipPath);
-  await spawnAsync('unzip', [ buildZipPath ], { cwd: BUILD_DIRECTORY });
+  let downloadedPercentage = 0;
+  await downloadFile(url, buildZipPath, (downloaded, total) => {
+    const percentage = Math.round(downloaded / total * 10) * 10;
+    if (percentage === downloadedPercentage)
+      return;
+    downloadedPercentage = percentage;
+    console.log(`Downloaded: ${downloadedPercentage}%`);
+  });
+
+  const zip = new AdmZip(buildZipPath);
```

Comment:
Don't we have extract-zip already for it as a dependency or does this do a better job?

Reply:
extract-zip only extracts; this also compresses & lists files.